airgun/views/organization.py

```diff
@@ -24,6 +25,7 @@ class OrganizationsView(BaseLoggedInView, SearchableViewMixin):
             'Actions': ActionsDropdown("./div[contains(@class, 'btn-group')]"),
         },
     )
+    dialog = Pf4ConfirmationDialog()
```

Comment:
I think this property (since its so generic and applies to all the components) can be defined at a higher(parent) class level than in a specific component class!

The comment is not blocker for this PR but can be added into future airgun improvements !

Reply:
@jyejare Agree. I can work on it in separate PR.